packages/paste-libraries/reakit/src/Tooltip/Tooltip.tsx

```diff
@@ -0,0 +1,78 @@
+import * as React from 'react';
+import {createComponent} from 'reakit-system/createComponent';
+import {createHook} from 'reakit-system/createHook';
+import {useForkRef} from 'reakit-utils/useForkRef';
+import {getDocument} from 'reakit-utils/getDocument';
+
+import {useDisclosureContent} from '..';
```

Comment:
issue (blocking): we should import these types from reakit directly, because right now there's a circular dependency.

Tooltip.tsx imports from `..` (index.tsx) and index.tsx imports from Tooltip.tsx.

This can cause issues

Reply:
Ahhh fair enough, I'll update this and all other occurrences where this is happening